app/src/components/AnimalList.js

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+
+import AnimalCard from "./AnimalCard";
+
+const AnimalList = ({ animals }) => {
+  let cards = animals.map((animal) => (
+    <AnimalCard key={animal.id} {...animal} />
+  ));
+
+  return (
+    <>
+      <h1>Cards</h1>
+      <div
+        style={{
+          overflowY: "scroll",
+          maxHeight: "300px",
```

Comment:
This is 1000% optional, but you could make the height responsive, ie `maxHeight: 50vh` so it'd take up 50% of the user's screen. When testing that out through dev tools on your site, that seems to be a good starter number. That way on larger screens like mine, users can view multiple cards at once :D See both Khaleesi and Osito at the same time. 

Not sure if this is just for me, but right now it always shows the scroll bar, even with that bigger height, but `overflowY: "auto"` seemed to just show the scroll bar when it was needed. 

Again this is all 1000% optional and you can even leave for later like Bill said!